ocaml-lsp-server/test/e2e/package.json

```diff
@@ -8,22 +8,18 @@
     "promote": "jest --updateSnapshot"
   },
   "dependencies": {
-    "jest-cli": "26.6.3",
```

Comment:
I don't think we need that explicit `jest-cli` dependency. It's already a [dependency of jest](https://github.com/facebook/jest/blob/main/packages/jest/package.json#L15) so it will get installed anyway